<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Impresoras</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Icono</normaloff>:/Icono</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Ingreso de Datos</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="lbMarca">
            <property name="text">
             <string>Marca:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="inMarca"/>
          </item>
          <item>
           <widget class="QLabel" name="lblModelo">
            <property name="text">
             <string>Modelo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inModelo"/>
          </item>
          <item>
           <widget class="QLabel" name="lblSerie">
            <property name="text">
             <string>Numero de serie:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inSerie">
            <property name="inputMask">
             <string>9999999</string>
            </property>
            <property name="maxLength">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Tipo:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="inColor">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="inBN">
               <property name="text">
                <string>B/N</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblContador">
            <property name="text">
             <string>Contador de Impresiones:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="inContador">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblPrecio">
            <property name="text">
             <string>Precio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inPrecio"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cantidad:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="inCantidad"/>
          </item>
          <item>
           <widget class="QPushButton" name="cmdIngresar">
            <property name="text">
             <string>Ingresar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Impresoras ingresadas</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="outImpresoras"/>
          </item>
          <item>
           <widget class="QLabel" name="lblTotal">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Total invertido:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="outTotal">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconAbrir</normaloff>:/iconAbrir</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/guardar</normaloff>:/guardar</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/salir</normaloff>:/salir</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconoInf</normaloff>:/iconoInf</iconset>
   </property>
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
